dbsync-central/application.prod.properties

```diff
@@ -0,0 +1,182 @@
+
+# Password of the openMRS datasource
+spring.openmrs-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the Management database *****************************************************
+#
+# Driver class of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.driverClassName=com.mysql.cj.jdbc.Driver
+
+# Dialect of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.dialect=org.hibernate.dialect.MySQLDialect
+
+# Url of the management datasource
+spring.mngt-datasource.jdbcUrl=jdbc:mysql://${openmrs.db.host}:${openmrs.db.port}/${dbsync_name}
+
+# User name of the management datasource
+spring.mngt-datasource.username=${spring_openmrs_datasource_user}
+
+# Password of the management datasource
+spring.mngt-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** ActiveMQ configuration ***********************************************************************
+#
+# Broker URL of the Artemis MQ broker shared with the receiver (SHOULD include the protocol e.g. tcp://192.168.1.2:61616)
+#spring.artemis.brokerUrl=tcp://artemis:61616
+#
+spring.artemis.host=${spring_artemis_host}
+spring.artemis.port=${spring_artemis_port}
+# User name of the Artemis broker
+spring.artemis.user=${spring_artemis_user}
+#
+# Password of the Artemis broker
+spring.artemis.password=${spring_artemis_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the ArtemisMQ endpoint to read from inbound DB sync payloads ****************
+# Use either one of the following confs. JMS endpoint if activeMQ is configured or file endpoint for TESTING purposes
+# Camel JMS Endpoint e.g. activemq:topic:<topic-name> or FILE Endpoint e.g. file:<folderPath> Should be absolute path
+camel.input.endpoint=activemq:topic:openmrs.sync.topic?subscriptionDurable=true&durableSubscriptionName=DB-SYNC-RECEIVER&connectionFactory=activeMqConnFactory&acknowledgementModeName=CLIENT_ACKNOWLEDGE&messageListenerContainerFactory=customMessageListenerContainerFactory
+activemq.clientId=DB-SYNC-REC
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Complex obs data sync configuration ***************************************************************
+#
+# Folder path where to write complex obs data files, should be the OpenMRS complex obs data directory
+camel.input.endpoint.complex.obs.data.directory=${eip.home}${file.separator}.OpenMRS${file.separator}complex_obs
```

Comment:
We should possibly remove the complex obs routes that way we remove this property, see https://jira.fgh.org.mz/browse/EC-93

Reply:
Ok